custom/ilsgateway/api.py

```diff
@@ -382,16 +382,14 @@ def web_user_sync(self, ilsgateway_webuser):
                 # if user exists in db it means he was already migrated but he changed email in v1
                 old_email = ils_sql_web_user.email
                 user = WebUser.get_by_username(old_email)
-                ils_domains = ILSGatewayConfig.get_all_enabled_domains()
-                # make sure it's user migrated from ILS and username is available
-                if all([domain in ils_domains for domain in user.domains])\
-                        and not WebUser.get_by_username(email):
```

Comment:
We can get rid of this check because we change only domain specific information and we have user existence check above:  https://github.com/dimagi/commcare-hq/pull/9939/files#diff-b02b2f2a345d9ec71be3b00c77ad6de0L375